inject/google/style/calendar.scss

```diff
@@ -6,4 +6,7 @@
 
 body * {
   font-family: var(--font), "Google Sans", Roboto, Arial, sans-serif, 'Material Icons Extended' !important;
+}
+.google-material-icons {
```

Comment:
It doesn't fix the problem completely. There is a span tag inside some of the use cases. You should include `.google-material-icons *` as well.
And this PR is old and needs to be updated with the new changes.